openpype/modules/deadline/plugins/publish/submit_fusion_deadline.py

```diff
@@ -221,11 +222,30 @@ def process(self, instance):
             # submitter is on Windows.
             "PYTHONPATH",
             "OFX_PLUGIN_PATH",
-            "FUSION9_MasterPrefs"
+            "FUSION9_MasterPrefs",
```

Comment:
```suggestion
```
Pretty sure this isn't needed - this is managed differently by OpenPype nowadays.

I'd also argue that `OFX_PLUGIN_PATH` should likely be set through Application environments and Tools, etc instead of submitting along env var of the user?

Reply:
Removed unnecessary env vars.